test/disk_size_test.rb

```diff
@@ -285,7 +305,15 @@
       disk_size = Y2Storage::DiskSize.MiB(12) / 3
       expect(disk_size.to_i).to be == 12 / 3 * 1024**2
     end
-    it "should refuse another type" do
+    it "should accept division by a float" do
```

Comment:
I mean, sometimes, things are implemented because it is easier to keep them than to leave them out. But that does not mean they make good sense.
But it is not such a big deal.